symfony/webpack-encore-bundle/2.0/post-install.txt

```diff
@@ -2,4 +2,4 @@
 
   * Compile your assets: <fg=green>npm run dev</>
 
-  * Or start the development server: <fg=green>npm run watch</>
+  * Or start the development server: <fg=green>npm run dev-server</>
```

Comment:
I would vote for keeping `watch` as the default suggestion as it works out of the box while `dev-server` requires installing an additional dependency.

Reply:
Why not

Reply:
in fact, I've also added `npm run build`, there is no reason to only show `npm run dev` and `npm run watch` (or `npm run dev-server`), since there is 4 scripts in the `package.json`.